jgrapht-core/src/main/java/org/jgrapht/alg/flow/min_cost/package-info.java

```diff
@@ -0,0 +1,4 @@
+/**
+ * Algorithms for minimum cost flow
+ */
+package org.jgrapht.alg.flow.min_cost;
```

Comment:
I think it is reasonable to assume that other algorithms for this problem will be implemented in the future, perhaps even by me, also I'm working maximum cost matchings at the moment. In fact, there are algorithms, that outperform this one, but they are much harder to implement. Probably, it can become another GSoC project for this year =)

Reply:
Alright sounds good, lets keep it in this package for now.